cmd/juju/application/config.go

```diff
@@ -406,7 +407,22 @@ func (c *configCommand) getConfig(client applicationAPI, ctx *cmd.Context) error
 	if len(results.ApplicationConfig) > 0 {
 		resultsMap["application-config"] = results.ApplicationConfig
 	}
-	return c.out.Write(ctx, resultsMap)
+
+	err = c.out.Write(ctx, resultsMap)
+	if err != nil {
+		return errors.Trace(err)
+	}
+
+	if featureflag.Enabled(feature.Generations) {
+		gen, err := c.ModelGeneration()
+		if err != nil {
+			return errors.Trace(err)
+		}
+
+		ctx.Stdout.Write([]byte(fmt.Sprintf("changes will be targeted to generation: %s\n", gen)))
```

Comment:
I would out a "\n" at the beginning too so that this line isn't abutting the config markup.

Reply:
done